scripts/generators/beats.py

```diff
@@ -62,6 +62,14 @@ def fieldset_field_array(source_fields, df_whitelist, fieldset_prefix):
         if not ecs_field['flat_name'] in df_whitelist:
             beats_field['default_field'] = False
 
+        # Include the `path` attribute for `alias` types.
+        # https://github.com/elastic/ecs/issues/876
+        if ecs_field['type'] == 'alias':
+            if 'path' in ecs_field:
+                beats_field['path'] = ecs_field['path']
+            else:
+                raise ValueError(f'The [path] property must be specified for field [{contextual_name}]')
```

Comment:
Since you've added the check in `schema.cleaner.field_mandatory_attributes` looking for `path` when `type: alias`, I don't think we have to worry about checking again here.